Lib/pickle.py

```diff
@@ -1913,7 +1913,7 @@ def _loads(s, /, *, fix_imports=True, encoding="ASCII", errors="strict",
         description='display contents of the pickle files')
     parser.add_argument(
         'pickle_file',
-        nargs='*', help='the pickle file')
+        nargs='+', help='the pickle file')
     args = parser.parse_args()
     if not args.pickle_file:
```

Comment:
This one is no more needed.

Reply:
I also see that `-` allows to write to STDIN as well, so might well document this feature.

Reply:
Sorry, I didn't understand. What do you mean by `-`
Should I revert last changes?

Reply:
Oh, you mean diff line? so that means i have to back the original docs example with `[...]` syntax?

Reply:
The `pickle_file` can be a regular filename/filepath but it can also be `-`:

```py
        for fn in args.pickle_file:
            if fn == '-':
                obj = load(sys.stdin.buffer)
            else:
                with open(fn, 'rb') as f:
obj = load(f)
```

So we also need to document that `-` means reading from `stdin` (like `echo ... | python -m pickle -`). OTOH, now that `pickle_file` expects 1 or more arguments, the `if not args.pickle_file:` block is dead code so you can remove the if-else and only keep the else branch. And you then move `import pprint` after `import argparse`

Reply:
thanks for details

Reply:
I added description to paragraph about argument, what you think now?